concepts/observability-configurable-monitoring/05-concept.md

```diff
@@ -15,7 +15,7 @@ To assure that the scraping collector will scale with the targets to scrape, it
 
 **Push support will be provided via OTLP only via the central deployment**
 
-A central collector, which listens for OTLP metrics, will be part of the setup. The related service port can be used to push custom metrics of any kind. That collector will be auto-scaled depending on resource consumption. Also, that instance will be configurable via the telemetry operator's MetricPipeline resource.
+A central collector, which listens for OTLP metrics, will be part of the setup. This dedicated central collector runs independent from the pull-based daemonset as the criterias for scaling are different. The central deployment mainly will auto-scale on resource consumption related to the incoming OTLP traffic. The related service port can be used for pushing custom metrics of any kind. Also, that instance will be configurable via the telemetry-operators MetricPipeline resource. In future it might be used for other kind of signals like traces as well.
```

Comment:
```suggestion
A central collector, which listens for OTLP metrics, will be part of the setup. This dedicated central collector runs independently from the pull-based daemonset because the criteria for scaling are different. The central deployment will auto-scale on resource consumption related to the incoming OTLP traffic. The related service port can be used for pushing custom metrics of any kind. Also, that instance will be configurable with the telemetry operator's MetricPipeline resource. In the future, it might also be used for other kinds of signals (like traces).
```

- Do we really need the `mainly`?
- What do you mean with `that instance`?
- `telemetry operator's`: just guessing this is what you meant?